docs/reference/search/search-your-data/retrievers-reranking/semantic-reranking.asciidoc

```diff
@@ -0,0 +1,161 @@
+[[semantic-reranking]]
+=== Semantic reranking
+
+preview::[]
+
+[TIP]
+====
+This overview focuses more on the high-level concepts and use cases for semantic reranking. For full implementation details on how to set up and use semantic reranking in {es}, see the <<retriever,reference documentation>> in the Search API docs.
+====
+
+Rerankers improve the relevance of results from earlier-stage retrieval mechanisms.
+_Semantic_ rerankers use machine learning models to reorder search results based on their semantic similarity to a query.
+
+First-stage retrievers and rankers must be very fast and efficient because they process either the entire corpus, or all matching documents.
+In a multi-stage pipeline, you can progressively use more computationally intensive ranking functions and techniques, as they will operate on smaller result sets at each step.
+This helps avoid query latency degradation and keeps costs manageable.
+
+Semantic reranking requires relatively large and complex machine learning models and operates in real-time in response to queries.
+This is a powerful technique for improving search relevance that works equally well with keyword, semantic, or hybrid retrieval algorithms.
+
+The next sections provide more details on the benefits, use cases, and model types used for semantic reranking.
+The final sections include a practical, high-level overview of how to implement <<semantic-reranking-in-es,semantic reranking in {es}>> and links to the full reference documentation.
+
+[discrete]
+[[semantic-reranking-use-cases]]
+==== Use cases
+
+Semantic reranking enables a variety of use cases:
+
+* *Lexical (BM25) retrieveal results reranking*
+** Out-of-the-box semantic search by adding a simple API call to any lexical/BM25 retrieval pipeline.
+** Add semantic search capabilities on top of existing indices without reindexing, perfect for quick improvements.
+** Ideal for environments with complex existing indices.
+
+* *Semantic retrieval results reranking*
+** Improves results from semantic retrievers using ELSER sparse vector embeddings or dense vector embeddings by using more powerful models.
+
+* *General applications*
+** Supports automatic and transparent chunking, eliminating the need for pre-chunking at index time.
+** Provides explicit control over document relevance in retrieval-augmented generation (RAG) uses cases or other scenarios involving language model (LLM) inputs.
+
+Now that we've outlined the value of semantic reranking, we'll explore the specific models that power this process and how they differ.
+
+[discrete]
+[[semantic-reranking-models]]
+==== Cross-encoder and bi-encoder models for semantic reranking
```

Comment:
We only support cross-encoders today, so this section can just focus on those models.

Reply:
Save this for future iterations 😉